tests/cms/views/pages/test_page_model.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from django.test.client import Client
+from django.urls import resolve, reverse
+
+from integreat_cms.cms.models.pages.page_translation import PageTranslation
+from tests.conftest import EDITOR, MANAGEMENT, PRIV_STAFF_ROLES
+
+
+@pytest.mark.parametrize(
+    "login_role_user", PRIV_STAFF_ROLES + [MANAGEMENT, EDITOR], indirect=True
+)
+@pytest.mark.django_db
+def test_clean_content(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+) -> None:
+    client, role = login_role_user
```

Comment:
Ideally this test should only test that the `clean_content` function works, so I don't think it should send a post request. Instead, it could just call the `clean_content` function

Reply:
Good point. Is applied now :)